web/packages/shared/components/UnifiedResources/shared/utils.ts

```diff
@@ -14,133 +14,116 @@
  * limitations under the License.
  */
 
-import React from 'react';
-import { ResourceIconName } from 'design/ResourceIcon';
-
 import {
-  Icon,
   Application as ApplicationIcon,
   Database as DatabaseIcon,
   Kubernetes as KubernetesIcon,
   Server as ServerIcon,
   Desktop as DesktopIcon,
 } from 'design/Icon';
+import { ResourceIconName } from 'design/ResourceIcon';
 
 import { DbProtocol } from 'shared/services/databases';
 import { NodeSubKind } from 'shared/services';
 
 import {
-  UnifiedResourceKube,
-  UnifiedResourceNode,
+  UnifiedResourceItem,
   UnifiedResourceUi,
-  UnifiedResourceDatabase,
+  UnifiedResourceNode,
   UnifiedResourceApp,
-  UnifiedResourceUserGroup,
+  UnifiedResourceDatabase,
   UnifiedResourceDesktop,
-} from './types';
-
-export interface UnifiedResourceCard {
```

Comment:
Actually, it made sense to me to keep the the type here, collocated with functions that used it. https://www.totaltypescript.com/where-to-put-your-types-in-application-code 

I'd treat `types.ts` as the last resort option, for situations where type is imported by multiple files or importing a type would cause an import loop.
In our case, the type is used only in this file.

Reply:
After my new changes we do actually use this type in another place (in `ResourceViewProps`). Thanks for the suggestion! I'll keep this in mind next time I'm working with `types.ts` files